appveyor.yml

```diff
@@ -1,5 +1,8 @@
 image: Visual Studio 2022
 configuration: Release
+branches:
+  only:
+    - v1.0.x
```

Comment:
`skip_branch_with_pr` [1] sounds like it might be helpful, but reading through some comments it will execute a build for the first push [2]. The option used here limits the branches considered [3].

I don't know if there's a way to conditionally create artifacts. The deploy section [4] does allow restricting it to a branch, but I don't know where that's actually deploying to. I don't see artifacts as part of this project's release tag.

[1]: https://www.appveyor.com/docs/appveyor-yml/
[2]: https://help.appveyor.com/discussions/questions/18437-skip_branch_with_pr-setting-doesnt-work
[3]: https://www.appveyor.com/docs/branches/#white--and-blacklisting
[4]: https://www.appveyor.com/docs/deployment/github/

Reply:
How about we only build on tags by setting `skip_non_tags: true` ?
Also, the deploy sections is filled but nothing is set on the download page for the releases (we only have the tar.gz of the sources).
The doc for config is here https://www.appveyor.com/docs/deployment/github/ but it all seems fine.

Reply:
> How about we only build on tags by setting `skip_non_tags: true` ?

That sounds like it will work.

> Also, the deploy sections is filled but nothing is set on the download page for the releases (we only have the tar.gz of the sources). The doc for config is here https://www.appveyor.com/docs/deployment/github/ but it all seems fine.

It wasn't clear where those were deployed to. I'm guessing `APPVEYOR_REPO_TAG: true` limited the builds, but I'm not sure why the tags were missed (if I understand that flag correctly).

After https://github.com/AOMediaCodec/libavif/pull/1923 and the follow up work is completed we can just delete this file. The older binaries on Appveyor will be deleted after 90 days.